src/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ModulePlan from './ModulePlan';
+import moduleplan from './moduleplan.json';
+
+class Home extends React.Component {
+  render() {
+    var modules = moduleplan.degree_course.modules;
+    var maxSemester = 6;
```

Comment:
I probably don't understand.
I can't remove the Home Class and the variables, can I?